src/components/slider/slider.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import RcSlider from 'rc-slider';
+import { FlexProps, Flex } from 'rebass';
+import { useTheme } from 'emotion-theming';
+import { ITheme } from '../../theme/types';
+import Labeling from '../typography/labeling';
+import Value from '../typography/value';
+
+export interface Props extends Omit<FlexProps, 'css' | 'onChange'> {
```

Comment:
Most of the components have the props named `componentNameProps`. Would you consider updating the name to keep consistency? You could also get rid of the renaming in the `index` file.

Reply:
Can't agree on this one: naming this as `SliderProps` would pollute the export scope of the Quartz, causing cases when the IDE wants to import stuff from `quartz/dist/slider/slider` :(